2018-11-26-sip-minutes.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: sips
+title: SIP Meeting Minutes - November 1-3 2018
+
+partof: minutes
+---
+
+# Minutes
+
+The following agenda was distributed to attendees, please [click here](https://docs.google.com/spreadsheets/d/1JdTyAxqqKqXtiXnZBPTaesMrWk7lyTWna1VPL3N5KWo/edit?usp=sharing).
+
+
+Jorge Vicente Cantero and Darja Jovanovic were the Process Leads.
+
+
+## Date and Location
+The meeting took place on November 1-3 2018, during the working hours 9 AM - 5 PM CET, at EPFL in Lausanne, Switzerland as well as other locations.
+
+The meeting was not recorded.
+
+
+## Attendees
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Jorge Vicente Cantero ([@jvican](https://github.com/jvican)), Process Lead
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Triplequote
+* Josh Suereth ([@jsuereth](https://github.com/jsuereth)), Independent
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), Scala Center
+* Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+* Miles Sabin ([@milessabin](https://github.com/milessabin)), Independent
+* Darja Jovanovic ([@darjutak](https://github.com/darjutak)), Scala Center
+
+## Joined online
+
+* Eugene Burmako ([@xeno-by](https://github.com/xeno-by)), Twitter
+
+## Not present
+
+* Heather Miller ([@heathermiller](https://github.com/heathermiller)), CMU
+
+## Proceedings
+### Opening Remarks
+
+The SIP Committee gathered for the first time face-to-face, for an extensive 3-day SIP meeting. The main goals and achievements of these meetings were:
+
+- Understand better the upcoming changes to Scala; 
+- Agree about the role of the Committee in the Scala 2 to Scala 3 transition;
+- Outline an action plan within a set time-frame;
+- Other: Unanimously voted for Guillaume Martres to join the Committee.
+
+*Better understanding* was enabled by in depth presentations and Q&As with the EPFL Dotty team; *the Approach* was agreed upon the first day which resulted in creating “FAQs about Scala 3” (see below) and *the Action Plan* was outlined and is still under construction; sveveral issues were opened and a project plan Meta programming in Scala 3 has been developed (please see the list at the end of this document).
```

Comment:
quote `"Meta Programming in Scala 3"`? also, I don't see anywhere that this document is linked.